php/Terminus/Commands/SiteCommand.php

```diff
@@ -1799,7 +1799,13 @@ public function setPhpVersion($args, $assoc_args) {
 
     if (isset($assoc_args['env'])) {
       $env               = $site->environments->get($assoc_args['env']);
-      $params['choices'] = ['default' => 'default', 53 => '5.3', 55 => '5.5',];
+      $params['choices'] = [
```

Comment:
It would be better to put this assignment outside of the scope of the `if` block, as `$params['choices']` is also used in the `else` block.

However, this is optional, as the `else` block (run when --env is not provided) does not seem to do anything.